docs/README-migration.md

```diff
@@ -1048,6 +1017,7 @@ SDL_RWFromFP(void *fp, SDL_bool autoclose)
 }
 ```
 

```

Comment:
My personal preference is to return a negative value on error.
Isn't this more consistent with the other stdio functions?

Reply:
No, it's not.